### DISCUSSION TITLE
ghelper.exe does not 'open' in the foreground

### DISCUSSION THREAD

**Discussion by fibxnwa** [seerge/g-helper#170] [General]
Hello,
I'm unable to start g-helper after download. 
The file saves to App Data / Local / Temp / Temp 1_GHz

I don't now the technical description, but the file doesn't turn on for me. 

Why is it doing this, and what can I do to fix it?

**Comment by seerge** [comment]
Try to unzip it into some normal place. Like your Downloads folder and run from there.

**Comment by fibxnwa** [comment]
This is fixed. 

Simply double-clicking the zip file does not actually unzip it, I needed to right click and click "extract all". A simple mistake from someone new to PCs.

**Reply by seerge** [reply]
Great, or just drag and drop file somewhere - that does same thing :)
